model/train.py

```diff
@@ -161,22 +161,48 @@ def compute_weighted_accuracy(logits, targets, weights=None):
     normalizing_factor = weights.sum()
 
   return loss.sum(), normalizing_factor
+  
+  
+def compute_sentence_accuracy(logits, targets, weights=None):
+  """Compute sentence accuracy for log probs and targets.
+
+  Args:
+   logits: [batch, length, num_classes] float array.
+   targets: categorical targets [batch, length] int array.
+   weights: None or array of shape [batch, length]
+
+  Returns:
+    Tuple of scalar loss and batch normalizing factor.
+  """
+  if logits.ndim != targets.ndim + 1:
+    raise ValueError("Incorrect shapes. Got shape %s logits and %s targets" %
```

Comment:
Nit: you could use formatted string literals instead (https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals).